src/Renci.SshNet/Renci.SshNet.csproj

```diff
@@ -50,6 +50,7 @@
 
   <ItemGroup>
     <PackageReference Include="BouncyCastle.Cryptography" Version="[2.4.0,)" />
+    <PackageReference Include="System.Formats.Asn1" Version="[8.0.1,)" />
```

Comment:
This is part of the runtime on modern .NET, so it should only be needed for `net462`, `netstandard2.0` and `netstandard2.1`,

Reply:
Makes sense. Thanks for the suggestion. @mus65 @Rob-Hague

Reply:
> (and System.Buffers - yay ArrayPool)

I forgot that System.Memory already depends on System.Buffers. Not sure why the Asn1 package declares an explicit dependency on System.Buffers